### DISCUSSION TITLE
"Mute game while in the background" toggle

### DISCUSSION THREAD

**Discussion by lovelydumpling** [ikemen-engine/Ikemen-GO#975] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

My ADHD brain regularly has myself multitasking while working on IKEMEN stuff. Checking out Discord messages, browsing the web, etc. while the game sits in the background running til I go back to it. But the music and audio is blaring the entire time unless I go out of my way to mute it in Volume Mixer and unmute it when I go back, which isn't very conducive to rapid multitasking.

### Describe the solution you'd like

An option to turn on "Mute while in the background", either in the options menu or through a config file of some kind. Then when it's not the active window, it doesn't make sound. 

### Search terms

mute background sound music volume inactive window

**Comment by UrsoManso** [comment]
That's a much needed feature for me too. Pausing kinda does the trick but since I alt+tab a lot having the engine to automically mute would be more convenient.

**Comment by potsmugen** [comment, marked as answer]
Added this in my branch. Volume setting instead of mute toggle. Will push when we move on to 1.1.
